packages/gatsby-theme-project-portal/utils/search.js

```diff
@@ -0,0 +1,202 @@
+const lodash = require("lodash")
+const lunr = require("lunr")
+function createSearchIndex(searchNodes) {
+  // searchNodes = {allProject[], allGeneralPage[], siteUrl}
+  let documents = []
+  Object.keys(searchNodes).forEach((page) => {
+    if (page === "siteUrl") {
+      return
```

Comment:
What's going on here? I don't understand this line.

Reply:
Looks like you actually want 
```
function createSearchIndex(searchNodes, siteUrl)
```
at the top of the function

Reply:
What's the siteUrl used for, anyway? Can you not pass it and get the same effect?